examples/CERA_workflow.ipynb

```diff
@@ -23,10 +23,10 @@
    "outputs": [],
    "source": [
     "from datetime import datetime, timedelta\n",
-    "from matplotlib import pyplot\n",
     "\n",
     "import geopandas as gpd\n",
     "import numpy as np\n",
+    "from matplotlib import pyplot\n",
```

Comment:
I don't think the changes in these notebooks: 
 * `examples/CERA_workflow.ipynb`
 * `examples/USGS_by_id.ipynb`
 * `examples/USGS_data.ipynb`
 * `examples/coops_data.ipynb`

are relevant with this PR?

Reply:
I'm not sure, but I guess it has something to do with the linting tools we're using to enforce order, etc.